src/sentry/api/endpoints/project_release_setup.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+from sentry.api.bases.project import ProjectEndpoint, ProjectReleasePermission
+from sentry.models import Group, ReleaseCommit, Repository, Deploy
+from rest_framework.response import Response
+
+
+class ProjectReleaseSetupEndpoint(ProjectEndpoint):
+    permission_classes = (ProjectReleasePermission, )
+
+    def get(self, request, project):
+        """
+        Checks how set up a project is on releases.
```

Comment:
I don't understand what this says, haha.

Reply:
Lol fair- i'll make it clearer. It's meant to see what steps have been completed of the 4 required to be "fully" set up on releases.

Reply:
Could use `Get release setup progress for a project`.